fastlane/Fastfile

```diff
@@ -9,7 +9,7 @@ platform :ios do
   desc "Installs dependencies using Carthage"
   lane :dependencies do |options|
     action = options[:action]
-    carthage(use_binaries: false, command: action, cache_builds: true, platform: 'all')
+    carthage(use_binaries: false, command: action, cache_builds: true, platform: 'iOS')
```

Comment:
Limiting build to the platform in question, otherwise it will build every platform even though it's only using 1. The previous option of 'all' was wrong. I